events/law_events/bureaucracy.txt

```diff
@@ -0,0 +1,417 @@
+		# 	OPTION = 3
+		# }
+	}
+	option = {
+		# hire new bureaucrats
+		name = bureaucracy.1.e
+		trigger = {
+			enactment_setback_count < 2
+		}
+		highlighted_option = yes
+		add_enactment_setback = 1
+		add_enactment_modifier = {
+			name = modifier_training_bureaucrat_replacements
+		}
+		# discord_impact_execute = {
+		# 	OPTION = 4
+		# }
+	}
+}
+
+# Advance event - Meritocracy
+bureaucracy.2 = {
+	type = country_event
+	title = bureaucracy.2.t
+	desc = bureaucracy.2.d
+	flavor = bureaucracy.2.f
+	category = enactment
+	event_image = {
+		video = "africa_leader_arguing"
+	}
+	icon = "gfx/interface/icons/event_icons/event_portrait.dds"
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	duration = 3
+	cooldown = {
+		months = normal_modifier_time
+	}
+	trigger = {
+		has_law = law_type:law_hereditary_bureaucrats
+		OR = {
+			is_enacting_law = law_type:law_appointed_bureaucrats
+			is_enacting_law = law_type:law_elected_bureaucrats
+		}
+		NOT = {
+			has_modifier = meritocracy
+		}
+	}
+	immediate = {
+		currently_enacting_law = {
+			save_scope_as = current_law_scope
+		}
+		if = {
+			limit = {
+				is_enacting_law = law_type:law_appointed_bureaucrats
+			}
+			ig:ig_intelligentsia = {
+				save_scope_as = meritocracy_ig
+			}
+		}
+		if = {
+			limit = {
+				is_enacting_law = law_type:law_elected_bureaucrats
+			}
+			ig:ig_petty_bourgeoisie = {
+				save_scope_as = meritocracy_ig
+			}
+		}
+		ig:ig_landowners = {
+			save_scope_as = bureaucrat_class
+		}
+		# Added by Babylon Discord
+		# Option 1
+		discord_impact_load = {
+			OPTION = 1
+			VALUE = 0
+		}
+		discord_impact_modifier_enactment_chance_x_ig = {
+			OPTION = 1
+			CHANCE = 20
+			IG = scope:meritocracy_ig
+		}
+		# Option 2
+		discord_impact_load = {
+			OPTION = 2
+			VALUE = 0
+		}
+		discord_impact_modifier_approval_x_ig = {
+			OPTION = 2
+			APPROVAL_ADD = -2 # Actually -10% pol str. for the current bureaucrats, gets converted to -2 approval
+			IG = scope:bureaucrat_class
+		}
+		discord_impact_modifier_enactment_chance_x_ig = {
+			OPTION = 2
+			CHANCE = 10 # In favour of new bureaucrats
+			IG = scope:meritocracy_ig
+		}
+		# Option 3
+		discord_impact_load = {
+			OPTION = 3
+			VALUE = 0
+		}
+		discord_impact_modifier_approval_x_ig = {
+			OPTION = 3
+			APPROVAL_ADD = 2 # A +10% pol str. modifier for the new bureaucrats gets converted to 2 approval
+			IG = scope:meritocracy_ig
+		}
+		discord_impact_modifier_enactment_chance_x_ig = {
+			OPTION = 3
+			CHANCE = 10 # 10% chance in favour of new bureaucrats
+			IG = scope:meritocracy_ig
+		}
+		# Exit added by Babylon discord
+	}
+	cancellation_trigger = {
+		NOT = {
+			currently_enacting_law = scope:current_law_scope
+		}
+	}
+	option = {
+		# use this to pass the law
+		name = bureaucracy.2.a
+		default_option = yes
+		# enactment success
+		add_enactment_modifier = {
+			name = meritocracy
+			multiplier = 2
+		}
+		# discord_impact_execute = {
+		# 	OPTION = 1
+		# }
```

Comment:
remove commented out code?

Reply:
I want to pick up on it for next release!